.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,32 @@
+name: "Feature request"
+description: Suggest a new feature for the bot and/or website.
+labels: ["s: needs triage", "t: feature"]
+body:
+- type: input
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    placeholder: I have searched the existing issues.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What is the feature?
+    description: |
+      Please decribe here what feature you would like to see added to the relevant part of this project.
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Would you like to implement this yourself?
+    description: Please note that higher priority features made be added by maintainers.
+    multiple: false
+    options:
+      - Yes, I'd like to implement this
+      - No, someone else can implement it
```

Comment:
Can we make this a checkbox? Feels awkward

Reply:
No, if its a checkbox then they don't have to answer it. Its a small thing with the issue forms which annoys me, but you can only make checkboxes themselves required, not the whole question itself.

Reply:
Ah. Figures. And there doesn't appear to be a radio button option. Is there one? If there is we ought use it, otherwise this is good enough.